.env.example

```
@@ -1,19 +1,31 @@
+TELEGRAM_BOT_MODE=polling
 
-# Flask session secret (auto-generated if omitted, but should be set in prod)
-SESSION_SECRET=your_flask_session_secret_here
+# Reserved for future webhook deployment support.
+TELEGRAM_WEBHOOK_URL=
+TELEGRAM_WEBHOOK_SECRET=
 
-# Telegram Mini App URL (set to enable the Mini App menu button in the bot)
-# MINIAPP_URL=https://your-app.example.com/miniapp
+# Legacy override if you need to pin a non-standard Mini App URL.
+# MINIAPP_URL=https://your-app.example.com/miniapp
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)